packages/graphql/test/stubCloudTypes.ts

```diff
@@ -533,3 +535,50 @@ export const CloudQuery: MaybeResolver<Query> = {
     })
   },
 }
+
+type EventType = DebugNewRelevantRunBar_SpecsSubscription | DebugTestingProgress_SpecsSubscription
+
+export function createRelevantRunSpecChangeEvent (type: 'current', completed: number, total: number, scheduledToCompleteAt: string | null): DebugTestingProgress_SpecsSubscription
+
+// eslint-disable-next-line no-redeclare
+export function createRelevantRunSpecChangeEvent (type: 'current', completed: number, total: number): DebugTestingProgress_SpecsSubscription
+
+// eslint-disable-next-line no-redeclare
+export function createRelevantRunSpecChangeEvent (type: 'next', completed: number, total: number, scheduledToCompleteAt: string | null): DebugNewRelevantRunBar_SpecsSubscription
+
+// eslint-disable-next-line no-redeclare
+export function createRelevantRunSpecChangeEvent (type: 'next', completed: number, total: number): DebugNewRelevantRunBar_SpecsSubscription
+
+// eslint-disable-next-line no-redeclare
+export function createRelevantRunSpecChangeEvent (type: 'current' | 'next', completed: number, total: number, scheduledToCompleteAt: string | null = null): EventType {
+  const event: any = {
```

Comment:
`any` necessary here? Function has a return type this evidently conforms to

Reply:
There may be a better way to type this, but without `any` then other type errors happen below this